signer/remote/remote_test.go

```diff
@@ -1,8 +1,7 @@
-package remote
+package remote_test
```

Comment:
what do you prefer as a solution because the duplicated code for
NewHandlerFromSigner is pretty meh

On Tue, Nov 3, 2015 at 1:43 PM, Nick Sullivan <notifications@github.com>
wrote:

> In signer/remote/remote_test.go
> <https://github.com/cloudflare/cfssl/pull/399#discussion_r43811304>:
>
> > @@ -1,8 +1,7 @@
> > -package remote
> > +package remote_test
>
> Fair enough. Does codecov.io play nice with this trick?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudflare/cfssl/pull/399/files#r43811304>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>